docker/bin/msfconsole

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+# determine script path
+pushd $(dirname $(dirname $(dirname $(readlink $0)))) > /dev/null
```

Comment:
```readlink``` for BSD platforms has similar issues, see http://stackoverflow.com/questions/3572030/bash-script-absolute-path-with-osx for some context.

The options suggested there are fairly complex however.  It is sufficient to evaluate if ```realpath``` is available or ```msfconsole.rc``` is available relative from ```pwd``` then notify user to execute from the framework path if can't resolve for now.